### PR TITLE
Added link to legendmud.org client

### PR DESCRIPTION
**Pull request by Ner-f**
Added a link to the legendmud.org mud client.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by steventhorne** [comment]
Did you check that this looks good on all screen sizes?

You can use the dev tools (F12) and then the "Responsive Design Mode" button to test it out.

This is what it looks like in firefox, but it looks very similar in chrome.
image

**Comment by Ner-f** [comment]
> 

Looks good to me.

image

File: www/src/views/shared/header.ejs [hunk 1]
```diff
@@ -35,6 +35,9 @@
                    <a ng-repeat="theme in themes" class="dropdown-item" href="" ng-click="setTheme(theme)">{{::theme}}</a>
                 </div>
 			</li>
+			<li class="nav-item">
+				<a class="nav-link" href="http://legendmud.org:19999/" target="_blank">Play Now</a>
```
Comment (steventhorne) [inline www/src/views/shared/header.ejs]: When linking to external sites, in addition to `target="_blank"`, I also like to add `rel="noopener noreferrer"`. This is for SEO purposes.

**Review by steventhorne — CHANGES_REQUESTED** [review]
Looks good!

A couple of minor changes to the external link, though.

**Review by steventhorne — APPROVED** [review]

**Pull request merged**